Basic OOP/BasicOOP/Interface/MyFirefoxDriver.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+
+namespace Basic_OOP.BasicOOP.Interface
+{
+    public class MyFirefoxDriver : Driver
+    {
+        public MyFirefoxDriver()
+        {
+            FirefoxDriverService service = FirefoxDriverService.CreateDefaultService(@"C:\Users\AHETKA\source\repos\N-Ix study 1\N-Ix study 1\bin\Debug");
```

Comment:
This code will only work on your machine. You can use the environment variable to specify the path
But you can skip this comment as it is not related to the current task